[Feature Request] Build instructions for getting pop shell shortcuts

Hello, I am getting an error of not having pop-shell-shortcuts together with pop-shell on my system. How can I install this?

Thanks

![image](https://user-images.githubusercontent.com/49439044/78539090-4d3f3380-77f2-11ea-8c80-cc9fa8fccde7.png)

> Hello, I am getting an error of not having pop-shell-shortcuts together with pop-shell on my system. How can I install this?
> 
> Thanks
> 
> ![image](https://user-images.githubusercontent.com/49439044/78539090-4d3f3380-77f2-11ea-8c80-cc9fa8fccde7.png)

error: failed to run custom build command for `glib-sys v0.9.1`

Caused by:
  process didn't exit successfully: `/home/me/Projects/shell-shortcuts/target/release/build/glib-sys-6c2c6f075f4895cd/build-script-build` (exit code: 1)
--- stderr
`"pkg-config" "--libs" "--cflags" "glib-2.0" "glib-2.0 >= 2.42"` did not exit successfully: exit code: 1
--- stderr
Package glib-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `glib-2.0.pc'
to the PKG_CONFIG_PATH environment variable
Package 'glib-2.0', required by 'virtual:world', not found
Package 'glib-2.0', required by 'virtual:world', not found


warning: build failed, waiting for other jobs to finish...
error: build failed
make: *** [Makefile:57: target/release/pop-shell-shortcuts] Ошибка 101

--- a/README.md
+++ b/README.md
@@ -54,13 +54,13 @@ This will generate the `pop-shell-shortcuts` executable in the `target/debug` di
 Install with `make`.
 
 ```sh
-make install prefix=/usr
+make install
 ```
 
 Alternatively, the debug version can be installed.
 
 ```sh
-make install prefix=/usr DEBUG=1
+make install DEBUG=1
 ```
 
 An `uninstall` target is also provided to remove the installed application.

--- a/README.md
+++ b/README.md
@@ -54,19 +54,19 @@ This will generate the `pop-shell-shortcuts` executable in the `target/debug` di
 Install with `make`.
 
 ```sh
-make install
+sudo make install
 ```
 
 Alternatively, the debug version can be installed.
 
 ```sh
-make install DEBUG=1
+sudo make install DEBUG=1
 ```
 
 An `uninstall` target is also provided to remove the installed application.
 
 ```sh
-make uninstall
+sudo make uninstall
 ```
 
 ## Run

--- a/README.md
+++ b/README.md
@@ -3,3 +3,78 @@
 Application for displaying and demoing Pop Shell shortcuts.
 
 ![](screenshot.png)
+
+## Dependencies
+
+The following dependencies are required to build `shell-extensions`.
+
+* [cargo](https://packages.debian.org/stable/rust/cargo)
+* [rustc](https://packages.debian.org/stable/rust/rustc)
+* [libgtk-3-dev](https://packages.debian.org/stable/libdevel/libgtk-3-dev)
+
+On a debian-derived distribution, the required dependencies are easily installed with `apt`.
+
+```sh
+apt install cargo libgtk-3-dev rustc
+```
+
+## Build
+
+First, get the project's source code.
+
+```sh
+git clone https://github.com/pop-os/shell-shortcuts.git
+```
+
+The following `make` commands are run from the top-level of the source directory, so move to that directory.
+
+```sh
+cd shell-shortcuts
+```
+
+The project's sources are built with `make`.
+
+```sh
+make
+```
+
+Assuming nothing goes wrong, this will generate the `pop-shell-shortcuts` executable in the `target/release` directory.
+
+By default, the application is built for release.
+To build the binary with debug symbols, provide `DEBUG=1` with the `make` command.
+
+```sh
+make DEBUG=1
+```
+
+This will generate the `pop-shell-shortcuts` executable in the `target/debug` directory.
+
+## Install
+
+Install with `make`.
+
+```sh
+make install prefix=/usr
+```
+
+Alternatively, the debug version can be installed.
+
+```sh
+make install prefix=/usr DEBUG=1
+```
+
+An `uninstall` target is also provided to remove the installed application.
+
+```sh
+make uninstall
+```
+
+## Run
+
+Execute the `pop-shell-shortcuts` binary.
+Assuming the binary has been installed in your path, run it as usual.
+
+```sh
+pop-shell-shortcuts
+```
+

--- a/README.md
+++ b/README.md
@@ -77,4 +77,3 @@ Assuming the binary has been installed in your path, run it as usual.
 ```sh
 pop-shell-shortcuts
 ```
-

